### ISSUE TITLE
Documentation and example validation CI/CT/CD

### ISSUE THREAD

**Issue by rob-metalinkage** [w3c/sdw-sosa-ssn#54]
Continuous Integration, testing and deployment of the specification should include validation of examples and be driven directly by the embedded documentation in the normative model.  

The assumption is that a Re-spec compatible HTML format can be derived and integrated into the general template providing background information. 

NB separating the model into a separate repository will assist in deploying such tools.

**Comment by rob-metalinkage** [comment]
unblocked with #117

**Comment by rob-metalinkage** [comment]
@avillar recommendations on how to do this with the OGC tooling please, given the repo isnt a standard Building Block structure.

**Comment by rob-metalinkage** [comment]
This is now set up - at https://opengeospatial.github.io/ogcapi-sosa/build/tests/report.html#bblock-ogc.sosa.properties.spec-examples

there are a couple of minor validation errors creeping in later edits..

**Comment by rob-metalinkage** [comment]
I think we can close it - since the validation process is now available as per issue name - do we do separate issues per validation failure? Or new issue? Or part of the examples rework issue as a closing condition?

**Comment by dr-shorthair** [comment]
Good work @rob-metalinkage

**Comment by rob-metalinkage** [comment]
closed in mtg 11/12/24
